sklearn/feature_selection/univariate_selection.py

```diff
@@ -307,29 +312,29 @@ def info_gain(X, y, gl_method="max"):
     y : array-like, shape = (n_samples,)
         Target vector (class labels).
 
-    gl_method : string, globalization_method, one of "aver", "max", "sum"
+    gl_method : string
+        Globalization_method, one of "aver" (default), "max", "sum".
 
     Returns
     -------
     scores : array, shape = (n_features,)
 
-    See also:
-    ---------
+    See also
+    --------
     info_gain_ratio : Information Gain Ratio
 
-    References:
-    -----------
+    References
+    ----------
     .. [1] J.R. Quinlan. 1993. C4.5: Programs for Machine Learning. San Mateo,
     CA: Morgan Kaufmann.
 
-    .. [2] Y. Yang and J.O. Pedersen. 1997. A comparative study on feature
+    .. [2] `Y. Yang and J.O. Pedersen. 1997. A comparative study on feature
```

Comment:
I've not checked what we do elsewhere, but I would rather only hyperlink the article title.